telegram-bot-api/ClientManager.cpp

```diff
@@ -289,9 +290,17 @@ void ClientManager::get_stats(td::PromiseActor<td::BufferSlice> promise,
     sb << "\n";
     sb << "id\t" << bot_info.id_ << "\n";
     sb << "uptime\t" << now - bot_info.start_time_ << "\n";
-    sb << "token\t" << bot_info.token_ << "\n";
+    if (!parameters_->stats_hide_sensible_data_) {
+      sb << "token\t" << bot_info.token_ << "\n";
+    }
     sb << "username\t" << bot_info.username_ << "\n";
-    sb << "webhook\t" << bot_info.webhook_ << "\n";
+    if (!parameters_->stats_hide_sensible_data_) {
+        sb << "webhook\t" << bot_info.webhook_ << "\n";
```

Comment:
Ooops a space too much here, but that would be cleaned up with the json one, #23

Reply:
You can fix it here then rebase the another branch with upstream when this it's merged